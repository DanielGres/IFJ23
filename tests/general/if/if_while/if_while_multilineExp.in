// DOBRE
/* OUTPUT:
    a: 9
    a: 10
    a: 11
    a: 12
*/
var a = 8
var b = 12
if (a < b) {
    while (a < b) {
        a = a 
        + 1;
        write("a: ", a)
    }
}