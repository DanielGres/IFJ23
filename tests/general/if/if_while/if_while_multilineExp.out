a: 9
a: 10
a: 11
a: 12